12_MeshLoaders/main.cpp

```diff
@@ -416,6 +461,11 @@ class MeshLoadersApp final : public MonoWindowApplication, public BuiltinResourc
 		Camera camera = Camera(core::vectorSIMDf(0, 0, 0), core::vectorSIMDf(0, 0, 0), core::matrix4SIMD());
 		// mutables
 		std::string m_modelPath;
+
+		SAssetBundle m_currentBundle;
```

Comment:
grab only the geometry, see https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/207/files#r2266047038